### PR TITLE
Galactic Conquest fixes

### LINKED ISSUE #1270: Syndicate AI Can't Recieve or Transmitt on Syndicate Comms Frequencies
So Round ID: 1963 
I was Ascalon the Syndicate AI, the entire time I couldn't hear the Syndicate Comms even when I had a intercom directly north of me programmed to broadcast and recieve comms chatter from Syndicate Comms.

### LINKED ISSUE #1311: Syndicate Crew in PVP get named according to their character's species
## Description

When you spawn in as a Syndicate agent on the PVP ship, you get a name. Obviously everyone needs a name, but since Syndicate agents are always humans it can get a bit wacky...
If you've selected a non-human species for your _normal character_ whoever you spawn in as on the Syndicate ship will get an auto-generated name based on the species you've selected, instead of just being a random human name, while still having a secondary last name for the Syndicate group you're a part of.

## Reproduction:

1. Set your species preference to a non-human (preferably moth or another race that has a recognizable random name pool).
2. Spawn in as a Syndicate agent **Roundstart** (not tested from lobby menu yet).
3. Examine yourself to see you now have a non-human name with a secondary last name which will be your collective Syndicate last name as chosen by the captain.

Image: Full name of a _very much human_ Syndicate Strategist, picked from the random moth name pool.
image

### LINKED ISSUE #1740: Mining issues with the Hammurabi
## Reproduction:
Starting mining with just the pickaxe is rather crippling, you are far slower than the NT can prep.
No conveyor switch on the furnace, you have to break the window.
No syndicate miner.
Syndicate space suit is inadequate for the long exposer to space that miner goes through, and will accumulate burn damage.
Travel distance from furnace to lathe is long, shrike was bad already but this is even worse.

### LINKED ISSUE #1742: Space on the SSV Hammurabi can't be clicked
## Testmerges:
#1726 
#1714 
#1692 
#1737 
#1730 
#1713 

## Reproduction:
1. Jump into space (accidentally or on purpose) on the Hammurabi
2. Try to throw something to stop you from floating off
3. Watch as you can't throw anything at space
4. Float to the edge of the map and get stuck instead of looping around.

### LINKED ISSUE #1815: Syndicate ships on PVP fail to initialize
## Round ID: Local tested
Every Syndicate PVP ship is broken, to put it simply it looks like viewing the map file on dream maker.
Probably something to do with initialization
Some things I noticed

Nothing can be clicked.
Guns can't be fired.
APCs are not named properly.
Everything is unpowered
Nothing is pressurized
Pipes and wires are visible on tiled floors
All airlocks are "falling apart"
Walls and windows are not smooth
Some closets have their items outside of them (eg Captain's locker)
Maintenance loot fails to spawn ("2 x maintenance loot spawner" appears instead)
And more.

## Testmerges:
#1720 

## Reproduction:
1. Run a PVP round either by changing pop requirements on pvp.dm or getting enough pop for PVP to roll
2. Observe the Syndicate ship.

### PR DESCRIPTION
**Pull request by someone543**
## About The Pull Request
This PR does the following:

**Initializes Syndicate ships:**
Beebase broke Syndicate ship initialization because initTemplateBounds got changed to a map_template proc instead of a parsed_map proc. This is fixed by adding a new proc called initParsedTemplateBounds, a slightly altered copy of proc/initTemplateBounds but for parsed_maps. This allows proc/instance_overmap to load interiors and space around Syndicate ships.
The proc instance_overmap will now make map borders for Syndicate ships work
The admin verb "Instance Overmap" has been fixed, allowing specific ships (All PVP ships and the Gunstar) to be added mid-round

**Overmap Changes**
During Galactic Conquest pre setup, the overmap gamemode will be changed to a special overmap gamemode also named Galactic Conquest. This is to prevent other overmap gamemodes such as boarding or delivery from happening, ensuring focus is on PVP. 
The notoriety system will be disabled during Galactic Conquest for obvious reasons.

**Syndicate Ship Changes:**
Hammurabi now has better mining equipment. Mining lockers and space suits can be found on deck 1 
Hammurabi CiC shutters button fixed
Hammurabi gauss dispenser has a crate
Hammurabi fighter console no longer requires ATC access
Hammurabi shell ammo sorter is now linked to the console
Hammurabi hanger areas altered. This fixes hanger airlocks
Hammurabi mag-cats has been renamed to Crimson 1 and Crimson 2. (Before both were Gold 2)
Hammurabi furnace conveyer belts now has a switch
Hammurabi kitchen has a slightly bigger and better 
Astraeus overmap ship name is now SSV Dark Astraeus instead of SSV Babylon
Astraeus gauss guns bound_x has been changed to allow people to get out without being stuck
Astraeus and Babylon AI cores now have working syndicate intercoms
Babylon now has a mining DRADIS console 
Babylon now has a ORM

**Other changes**
Syndicate cameras will now initialize properly, allowing the Syndicate camera console to be used.
A timer has been added to FTL navigation consoles initialization, fixing a issue where Syndicate navigation consoles would not work unless reconstructed.
Syndicate crew antag tips have been updated
Syndicate crew will no longer have non-human names 
Syndicate crew headsets are now locked to Syndicate frequency.
check_completion() will now check if a faction has over ~~1000~~ 700 points during PVP, fixing point victory.
Moves Hammurabi map files to Instanced map files
Nanopumps initialization addtimer has been increased to 30 seconds instead of 10. This should allow bigger ships such as the Babylon to link properly.

## Why It's Good For The Game
Allows a game mode to function properly.
fixes #1815
fixes #1740 
fixes #1742
fixes #1311 
fixes #1270 

## Changelog
:cl:
add: Hammurabi has been given better mining equipment (The furnace is still there)
add: Galactic Conquest now has its own overmap gamemode
add: Babylon now has a ORM
add: Syndicate intercoms added to Astraeus and Babylon
fix: PVP ships now properly initialize
fix: Admin verb Instance Overmap works again
fix: Syndicate camera console works again
fix: PVP point victory fixed
fix: Hammurabi CIC shutter button fixed.
fix: Astraeus gauss turrets will no longer trap people 
fix: Syndicate crew will no longer have non-human names
tweak: Babylon's helmsman DRADIS has been replaced with a mining DRADIS
tweak: Galactic Conquest antag tips updated
tweak: Notoriety system disabled during PVP
tweak: PVP Astraeus ship name to SSV Dark Astraeus
tweak: Hammurabi hanger areas changed to fix airlocks
tweak: Hammurabi mag-cats name changed to Crimson 1 and Crimson 2
tweak: Hammurabi map files have been moved to Instanced map files
tweak: Hammurabi kitchen is slightly bigger and better
tweak: Nanopumps initialization timer is now 30 seconds
tweak: Syndicate headsets are now locked to Syndicate frequency
code: Overmap mode setup will now come after gamemode pre_setup. 
code: Added proc initParsedTemplateBounds, a modified copy of initTemplateBounds but for parsed maps
/:cl:

### CODE REVIEW

**Comment by covertcorvid** [comment]
I can address the "Syndicate Sabres can jump to the main ship" RP in #1797

**Comment by covertcorvid** [comment]
Wait we actually **were** using that variable? How'd mine compile then... Oh well. I can put it back.

File: nsv13/code/game/gamemodes/overmap/galactic_conquest.dm [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+//Acquire 1000 tickets for NT
+/datum/overmap_gamemode/galactic_conquest
+	name = "Galactic Conquest"
+	desc = "PVP Gamemode - Acquire 1000 victory tickets for Nanotrasen or destroy the Syndicate Flagship"
+	brief = "Intel suggests that the Syndicate are mounting an all out assault on the Sol sector. Patrol this area and eliminate any Syndicate forces with extreme prejudice."
+	starting_system = "Argo"
+	starting_faction = "nanotrasen"
+	objective_reminder_setting = 1
+	objective_reminder_interval = 2 HOURS //No external pressure in this round...
+	selection_weight = 0 //This overmap gamemode will be used only during PVP
```
Comment (Kenionatus) [inline nsv13/code/game/gamemodes/overmap/galactic_conquest.dm:12]: Aaaaaaa. Why do so many people not use doc comments?

File: nsv13/code/game/gamemodes/pvp/items.dm [hunk 2]
```diff
@@ -431,3 +431,21 @@
 	network = list("syndicate")
 
 /obj/machinery/camera/syndicate/autoname
+	var/number = 0
```
Comment (Bokkiewokkie) [inline nsv13/code/game/gamemodes/pvp/items.dm:434]: why not just make the default value 1?

File: nsv13/code/modules/overmap/ai-skynet.dm [hunk 3]
```diff
@@ -2239,7 +2241,7 @@ Seek a ship thich we'll station ourselves around
 		return FALSE
 
 	var/list/choices = flist("_maps/map_files/Instanced/")
-	var/ship_file = input(usr, "What ship would you like to load?","Ship Instancing", null) as null|anything in choices
+	var/ship_file = file("_maps/map_files/Instanced/"+input(usr, "What ship would you like to load?","Ship Instancing", null) as null|anything in choices)
```
Comment (Bokkiewokkie) [inline nsv13/code/modules/overmap/ai-skynet.dm:2244]: We should also move the Hammurabi files to this folder.

**Review by Bokkiewokkie — APPROVED** [review]
Based

File: nsv13/code/game/gamemodes/pvp/items.dm (hunk 2 above)
Comment (someone543) [reply nsv13/code/game/gamemodes/pvp/items.dm:434]: Changing that value to one causes areas with multiple cameras to be named area 2, area 3, area 4. skipping 1.

Comment (Bokkiewokkie) [reply nsv13/code/game/gamemodes/pvp/items.dm:434]: Ah I didn't see the part where it did `number +1` in the code (which is honestly pretty fucked up but alright)

**Comment by someone543** [comment]
Things to fix:
Stacked consoles on Hammurabi

**Review by Bokkiewokkie — APPROVED** [review]

**Pull request merged**